.github/workflows/test_on_push.yaml

```diff
@@ -0,0 +1,21 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  run_tests:
+    strategy:
+      matrix:
+        tarantool-version: ["1.10", "2.3", "2.4", "2.5"]
```

Comment:
I'd recommend to always use `fail-fast: false` strategy, so all invoked job runners would be given chance to complete, regardless of failure of otehr runners